lib/bap/bap_inject_externals.mli

```diff
@@ -0,0 +1,9 @@
+open Bap_types.Std
+open Bap_image_std
+open Bap_sema.Std
+open Bap_disasm_std
+
+(** [run prg insn spec] performs search for relocations in [spec] and
+    add inject synthetic subroutines for each external function. Also
```

Comment:
"add inject" -> inserts or injects

I'm not sure, why you're using word injection for this procedure at all, for me injection is an injective mapping, i.e., when you map one set to another without losing information.